.github/workflows/build.yml

```diff
@@ -0,0 +1,123 @@
+name: Build and upload to PyPI
+# publish when a (published) GitHub Release is created
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels:
+    name: Build wheels [${{ matrix.os }}, ${{ matrix.platform }}, py ${{ matrix.python-version }}]
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        platform: [x64]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Set up python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.6.3
+      - name: Install Visual C++ for Python 2.7
+        if: runner.os == 'Windows'
+        run: |
+          choco install vcpython27 -f -y
```

Comment:
Why do we need python 2.7?

Reply:
I got that from the `cibuildwheel` github action [example](https://cibuildwheel.readthedocs.io/en/stable/setup/). They were not using 2.7 in that example either. The just need the c compilers from visual studios so I don't think it matters which python version is there. 

I don't know much about compiling c codes on windows. Do you think its worth looking into to see if there is a better package?

Reply:
That is a really old compiler. The compilers are installed as part of the base image so I am pretty sure you can delete that bit and everything will JustWork

Reply:
looks like you are correct. I removed it and things still work.